packages/plugin-assets/tests/init.test.js

```diff
@@ -25,6 +12,19 @@ jest.mock('@twilio-labs/serverless-api/dist/api/environments', () => {
   };
 });
 
+const { init } = require('../src/init');
```

Comment:
Why was this moved, do we have to mock before importing now?

Reply:
<img width="1122" alt="Screenshot 2024-03-18 at 12 03 03 PM" src="https://github.com/twilio-labs/serverless-toolkit/assets/33580233/e2f19ce2-e486-4b4a-b675-4ea5fe40f908">

automatic hoisting of mocks seems to be broken in v29, as it no longer works for es modules, I looked at the docs and found the above.